src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -58,6 +58,9 @@ public static ConversationStore getTestInstance(PersistentStorageAgent persisten
   /** The in-memory list of Conversations. */
   private List<Conversation> conversations;
 
+  /** The in-memory count for conversations. */
+  private int conversationCount;
```

Comment:
Does this value ever deviate from conversations.size()?  If not it doesn't seem necessary.